meta-resin-common/recipes-containers/docker-disk/files/entry.sh

```diff
@@ -1,18 +1,16 @@
 #!/bin/bash
 
-#--- This script is now setup to fail if any environment variable is missing.
-
 set -o errexit
 set -o nounset
 
 DOCKER_TIMEOUT=20 # Wait 20 seconds for docker to start
 
+# Default values
 export PARTITION_SIZE=${PARTITION_SIZE:=1000}
-export TARGET_REPOSITORY=${TARGET_REPOSITORY:=resin/i386-supervisor}
-export TARGET_TAG=${TARGET_TAG:=master}
+export TARGET_REPOSITORY=${TARGET_REPOSITORY:=}
+export TARGET_TAG=${TARGET_TAG:=}
```

Comment:
Indeed. But I tried to no change but restructure. So I think this should be address in a later PR.